modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -1,249 +1,272 @@
+If you do not do this, you might be violating your terms with {redhat}.
+====
 
+== Server requirements
 
-== Set up a {rhel} Client
-
-Ensure that your client meets these requirements before you start:
-
-* 8{nbsp}GB RAM or more
-* Two or more physical or virtual CPU cores
-* Access to {rhel} installation and subscription media
-* An LVM or an NFS mount is recommended
-
-You will need to ensure you provision enough disk space.
-The [path]``/var/spacewalk`` directory contains all mirrored RPMs,  and can take a significant amount of disk space.
-For example, the {rhel}{nbsp}6 x86_64 channels require over 90{nbsp}GB.
+Ensure that your {productname} Server meets the requirements outlined at xref:installation:hardware-requirements.adoc[].
 
 Taskomatic will use one CPU core, and requires at least 3072{nbsp}MB RAM.
-To ensure that Taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+
+To ensure that Taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, add this line:
 
 ----
 taskomatic.java.maxmemory=3072
 ----
 
-[WARNING]
-====
-You are responsible for arranging access to Red Hat base media repositories and RHEL installation media.
-You must obtain support from either Red Hat or {suse} for all your RHEL systems.
-If you do not do this, you might be violating your terms with Red Hat.
-====
-
+Restart Taskomatic:
+----
+systemctl restart taskomatic
+----
 
+== Import {redhat} Entitlement and CA Certificate
 
-== {rhel} Channel Management
+During the next few paragraphs, you will learn how to import the {redhat} CA and entitlement certificate.
 
-The base {rhel} software channel does not contain any packages.
-This is because {suse} does not provide {rhel} base media.
-You will need to obtain base media from Red Hat, which you can then add as a child channel to the {rhel} parent channel.
+[WARNING]
+====
+Entitlement certificates have embedded expiration dates tied to the length of the support subscription. You must repeat this process with updated entitlement certificates as needed to avoid disruption in channel mirroring.
+====
 
-The {rhel} and tools channels are provided by {scc}.
-You can synchronize your client with the [command]``mgr-sync`` command to get them.
+==== Entitlement Certificate
 
-Because the {rhel} channels are particularly large, it can take up to 24 hours for an initial channel synchronization to complete.
-When you have completed the initial synchronization, we recommended you clone the channel before working with it.
-This provides you with a backup of the original synchronization data.
+{redhat} Subscription Manager is a local service which tracks installed products and subscriptions on a local system to help manage subscription assignments. To obtain your certificates, you must register your system using this subscription manager tool supplied by {redhat}.
 
```

Comment:
{redhat} Your system may be registered to several different places - the Red Hat CDN directly, a Satellite server, or to a RHUI environment, such as a public cloud.  Make sure that the URL you choose in the repository is associated with your entitlement.  In other words, a Satellite-registered machine will only have entitlements for Satellite repositories from subscription-manager, so those certificates will not enable you to connect directly to the Internet-based CDN repositories.